.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+	"parserOptions": {
+		"ecmaVersion": 6,
+		"sourceType": "module",
+		"ecmaFeatures": {
+			"jsx": true
+		}
+	},
+	"rules": {
+		"indent": ["error", 2, {
+			"VariableDeclarator":{
+				"var": 2,
+				"let": 2,
+				"const": 3
```

Comment:
What does the `3` stand for here, also I don't think I understand what the variable declarator is doing

Reply:
Required # of indents

Reply:
results in variables being lined up:

```
var a, // 2 indents
    b; // 2 indents

const A, // 3 indents
      B, // 3 indents
```

Reply:
@jwu910 so shouldn't var/let get 3 and const get 2 so that they are all lined up?
if what you are saying that it should look like
```
var     a;
let      b;
const C;
```
Or am I still not understanding this haha

Reply:
Its indents from the 1st column

if we did 2 2 2 for var, let, and const, it would look like this: 
```
var a,
    b;

let a,
    b,

const a,
    b;
```